distribution/changelog.txt

```diff
@@ -16,6 +16,8 @@
 - Fix: [#12845] Deleting ride with active ad campaign creates incorrect notification.
 - Fix: [#12857] Incorrect Peep thoughts in imported RCT1 parks.
 - Fix: Incomplete loop collision box allowed overlapping track (original bug).
+- Change: [#12806] Add Esperanto diacritics to the sprite font.
```

Comment:
@AaronVanGeffen I have always put new letters in as a "feature". I suppose that "improved" would also fit, but we tend to use "Change" for stuff that is more likely to be neutral or controversial.

Reply:
@Gymnasiast Fair enough; I think "Improved" would work just as well. Feel free to address this as you see fit. I just didn't want it to be forgotten.